src/RetrySettings.php

```diff
@@ -349,6 +379,26 @@ public function with(array $settings)
         return new RetrySettings($settings + $existingSettings);
     }
 
+    /**
+     * Creates an associative array of the {@see Google\ApiCore\RetrySettings} timeout fields configured
+     * with the given timeout specified in the $timeout parameter interpreted as a logical timeout.
+     *
+     * @param int $timeout {
```

Comment:
Just a minor nit, the bracket style is used for notating the elements in an array, this can just be `@param int $timeout desc`

Reply:
Thanks! Done.